Lib/test/test_tools/test_i18n.py

```diff
@@ -517,6 +517,219 @@ def test_parse_keyword_spec(self):
+                  b'\nContent-Type: text/plain; charset=UTF-8\nContent-' \
+                  b'Transfer-Encoding: 8bit\nPlural-Forms: nplurals=2' \
+                  b'; plural=(n > 1);\n\x00Bonjour !\x00{n} cheval\x00{n' \
+                  b'} chevaux\x00'
+    file2_fr_mo = b"\xde\x12\x04\x95\x00\x00\x00\x00\x03\x00\x00\x00" \
+                  b"\x1c\x00\x00\x004\x00\x00\x00\x00\x00\x00\x00\x00" \
+                  b"\x00\x00\x00\x00\x00\x00\x00L\x00\x00\x00\x06\x00" \
+                  b"\x00\x00M\x00\x00\x00\n\x00\x00\x00T\x00\x00\x00[" \
+                  b"\x01\x00\x00_\x00\x00\x00\r\x00\x00\x00\xbb\x01" \
+                  b"\x00\x00\x0f\x00\x00\x00\xc9\x01\x00\x00\x00Bye.." \
+                  b".\x00It's over.\x00Project-Id-Version: python 3.8" \
+                  b"\nReport-Msgid-Bugs-To: \nPOT-Creation-Date: 2018" \
+                  b"-11-30 23:57+0100\nPO-Revision-Date: 2018-11-30 2" \
+                  b"3:57+0100\nLast-Translator: s-ball <s-ball@lapost" \
+                  b"e.net>\nLanguage-Team: French\nLanguage: fr\nMIME" \
+                  b"-Version: 1.0\nContent-Type: text/plain; charset=" \
+                  b"UTF-8\nContent-Transfer-Encoding: 8bit\nPlural-Fo" \
+                  b"rms: nplurals=2; plural=(n > 1);\n\x00Au revoir ." \
+                  b"..\x00C'est termin\xc3\xa9.\x00"
+
+    # image of merging both po files keeping second header
+    file12_fr_mo = b"\xde\x12\x04\x95\x00\x00\x00\x00\x05\x00\x00\x00" \
+                  b"\x1c\x00\x00\x00D\x00\x00\x00\x00\x00\x00\x00\x00" \
+                  b"\x00\x00\x00\x00\x00\x00\x00l\x00\x00\x00\x06\x00" \
+                  b"\x00\x00m\x00\x00\x00\x06\x00\x00\x00t\x00\x00\x00" \
+                  b"\n\x00\x00\x00{\x00\x00\x00\x14\x00\x00\x00\x86" \
+                  b"\x00\x00\x00[\x01\x00\x00\x9b\x00\x00\x00\r\x00" \
+                  b"\x00\x00\xf7\x01\x00\x00\t\x00\x00\x00\x05\x02\x00" \
+                  b"\x00\x0f\x00\x00\x00\x0f\x02\x00\x00\x16\x00\x00" \
+                  b"\x00\x1f\x02\x00\x00\x00Bye...\x00Hello!\x00It's " \
+                  b"over.\x00{n} horse\x00{n} horses\x00Project-Id-Ver" \
+                  b"sion: python 3.8\nReport-Msgid-Bugs-To: \nPOT-Crea" \
+                  b"tion-Date: 2018-11-30 23:57+0100\nPO-Revision-Date" \
+                  b": 2018-11-30 23:57+0100\nLast-Translator: s-ball <" \
+                  b"s-ball@laposte.net>\nLanguage-Team: French\nLangua" \
+                  b"ge: fr\nMIME-Version: 1.0\nContent-Type: text/plai" \
+                  b"n; charset=UTF-8\nContent-Transfer-Encoding: 8bit" \
+                  b"\nPlural-Forms: nplurals=2; plural=(n > 1);\n\x00A" \
+                  b"u revoir ...\x00Bonjour !\x00C'est termin\xc3\xa9." \
+                  b"\x00{n} cheval\x00{n} chevaux\x00"
+    def imp(self):
+        i18ndir = os.path.join(toolsdir, 'i18n')
+        sys.path.append(i18ndir)
+        import msgfmt
+        sys.path.remove(i18ndir)
+        return msgfmt
```

Comment:
Is it really necessary to import the script as a module? I thought it was only a command-line tool.